.gitignore

```diff
@@ -1,2 +1,4 @@
 configs.ts
 *.dylib
+
+configs.example.ts
```

Comment:
The example file should not be ignored. You should create a new file `configs.ts` where u have ur private configs. The example file is used for new devs to see what their configs.ts file needs.